docs/Server-Setup.md

```diff
@@ -127,7 +127,7 @@ To run the tests against an instance that you've already set up:
 
 Or to run a specific test (e.g., basic):
 
-    docker exec codalab_rest-server_1 python test_cli.py basic
+    docker exec codalab_rest-server_1 python3 test_cli.py basic
```

Comment:
Do we need `python3.6` here? Or is the default version of `python3` already 3.6 on the docker containers?

Reply:
Python3 currently points to 3.5 in the rest-server container. There was a ticket to point Python3 version in Dockerfile.default-cpu && Dockderfile.default-gpu. Do we need to point Python3 to 3.6 as well in rest-server container?